src/types.db

```diff
@@ -148,6 +149,9 @@ node_octets             rx:DERIVE:0:U, tx:DERIVE:0:U
 node_rssi               value:GAUGE:0:255
 node_stat               value:DERIVE:0:U
 node_tx_rate            value:GAUGE:0:127
+nr_overcommit_hugepages value:DERIVE:0:U
+nr_hugepages            value:DERIVE:0:U
```

Comment:
Hi @maryamtahhan, I'm suggesting to report the used pages *only*. This allows to create a "stacked" graph which sums up nicely to a (usually constant) total. This is what the *memory* and *df plugins* are doing, for example.

When reporting the "total" too, that doesn't work because some bytes are accounted for twice or more times, e.g. free bytes would be included in "free" as well as "total".

Best regards,
—octo